inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details_decision_authors.html

```diff
@@ -1,4 +1,7 @@
 <div ng-if="vm.record._extra_data._action === 'author_approval' && vm.record._workflow.status === 'HALTED'">
+<button class="btn btn-info"
+        ng-click="Utils.redirectToReviewSubmission(vm.record.id)">Review Submission
```

Comment:
I am wondering if we should have a `Utils.redirect` function and in this template that is specific to the author forms you could do `ng-click="Utils.redirect('/submit/author/newreview?objectid={{vm.record.id}})"`